playground/azuresql/AzureSql.AppHost/Program.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddAzureProvisioning();
+
+var rg = builder.AddParameter("rg");
+
+var db = builder.AddBicepTemplate("sql1", "sql.bicep")
+    .WithParameter("p0", rg);
+
+builder.AddProject<Projects.AzureSql_ApiService>("api")
+       .WithEnvironment(ctx =>
+       {
+           var databaseName = "db";
+
+           if (ctx.PublisherName == "manifest")
+           {
+               ctx.EnvironmentVariables["ConnectionStrings__db"] = $"Server=tcp:{{{db.Resource.Name}.outputs.sqlServerName}}.database.windows.net,1433;Initial Catalog={databaseName};Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;Authentication=\"Active Directory Default\";";
```

Comment:
We need to think about how we use Bicep outputs. In ARM deployments, outputs can't be flagged as secrets. For resource types that support managed identity auth that isn't such an issue because the connection string doesn't contain any real secrets, but it can be an issue for those resource types that don't support managed auth.

Reply:
Are you talking about this low level primitive or something else (higher level resource types)?